js/client/modules/@arangodb/testutils/process-utils.js

```diff
@@ -732,12 +732,14 @@ function executeAndWait (cmd, args, options, valgrindTest, rootDir, coreCheck =
     };
   }
 
-  let instanceInfo = {
-    rootDir: rootDir,
-    pid: 0,
-    exitStatus: {},
-    getStructure: function() { return {}; }
-  };
+  if (!instanceInfo.hasOwnProperty('rootDir')) {
+    instanceInfo = {
```

Comment:
If `instanceInfo` was set by the caller, all data in it will be overridden here.

Reply:
`rootDir` is a mandatory attribute of `instanceInfo`.  if its not there, its safe to overwrite it.

Reply:
Just saying that you are overwriting _everything_ that the caller has set here, not just `rootDir`. 
I can live with that, just wanted to mention it.